Game/RainMeadow.EntityHooks.cs

```diff
@@ -284,162 +284,88 @@ private void OverWorld_WorldLoaded(On.OverWorld.orig_WorldLoaded orig, OverWorld
 
                 Room room = null;
 
-                if (true)
```

Comment:
git really didn't like the indentation change, is this all just removing the if(true) and braces?

Reply:
yeah, there's an additional change for changedRegion.